src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -242,6 +277,12 @@ private void invokeCallbackEndpoint(
             exceptionRecordData.put("searchCaseReference", searchCaseReference);
         }
 
+        if (containPayments) {
+            exceptionRecordData.put("containsPayments", "Yes");
+            exceptionRecordData.put("envelopeId", "21321931312-32121-312112");
+            exceptionRecordData.put("poBoxJurisdiction", "sample jurisdiction");
+        }
+
```

Comment:
not sure why we need this to be added to exception record ?

Reply:
we publish envelopeId and jurisdiction.